From f42f80b39f45e64051e3bc71b90592703f1b8ed1 Mon Sep 17 00:00:00 2001
From: Ole Petter <ole.orhagen@northern.tech>
Date: Thu, 14 Oct 2021 13:29:22 +0200
Subject: [PATCH] Instrument for coverage collection

Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>
---
 main.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.go b/main.go
index cd5b9e8..4726af4 100644
--- a/main.go
+++ b/main.go
@@ -15,7 +15,6 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
 	"github.com/mendersoftware/mender-artifact/cli"
@@ -26,6 +25,7 @@ func run() error {
 	if err != nil {
 		fmt.Println(err)
 	}
+	coverReport()
 	return err
 }
 
-- 
2.33.0

